softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/implementation/AbstractHandler.java

```diff
@@ -0,0 +1,38 @@
+package com.softgroup.common.router.api.implementation;
+
+import com.softgroup.common.datamapper.DataMapper;
+import com.softgroup.common.protocol.Request;
+import com.softgroup.common.protocol.Response;
+import com.softgroup.common.protocol.ResponseStatusEnum;
+import com.softgroup.common.router.api.interfaces.Handler;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.Serializable;
+import java.util.Map;
+
+/**
+ * Created by user on 08.04.2017.
+ */
+public abstract class AbstractHandler<W extends Serializable,V> implements Handler {
+    Class<V> typeRequestData;
+    @Autowired
+    DataMapper dataMapper;
+
+    public AbstractHandler(Class<V> typeRequestData) {
```

Comment:
@werbower Use some one style. Or constructor injection or field injection. Both are not allowed.